src/libs/Navigation/Navigation.js

```diff
@@ -89,7 +95,9 @@ function dismissModal(shouldOpenDrawer = false) {
     });
 
     // Navigate back to where we were before we launched the modal
-    goBack();
+    if (canGoBack) {
```

Comment:
Can we maybe just return early here if it's a drawer actually? I don't think we need `canGoBack` at all.

```js
let isLeavingDrawerNavigator;

...

if (state.type === 'drawer') {
    isLeavingDrawerNavigator = true;
    return StackActions.pop();
}

...

if (isLeavingDrawerNavigator) {
    return;
}
```

I think this would be clearer since we're creating a weird requirement to add another parameter when calling this method and I don't think we need to.